### PR TITLE
Badly merged pr gone wrong, gone sexual, gone git [git reset --hard]

### PR DESCRIPTION
**Pull request by Kmc2000**
Credit: Injazz ^^^

Updates the bridge's look to something a bit more space shipey by:

Updating the viewscreen
Adding snowflake chairs

https://drive.google.com/file/d/0B9VUbibptgF2MlZuWGlaalc4QlE/view?usp=sharing
Demo which shows off EVERYTHING ^^

NOTE: computers look a lot less weird than they do in that now, fixed.

Examples:

BEFORE:

AFTER:

https://i.gyazo.com/5522d8b2956d00601cb3df723f6b9a87.mp4

Also made the shieldgen calm the fuck down @Ecolli you can stop nagging me now

##Changeling##

:cl: Kmc2000
add: Added a new type of bridge airlock
add: Added some new uniforms, check them out under the misc category in cargo (when they're in stock)
tweak: made the shieldgen less seizure inducing
imageadd: Added new chairs for the bridge, made a better viewscreen and the captain now has his own chair
tweak: updated the old computer sprite, it now has directions!
experiment: added a subtype to the bridge computers for when i make a snowflake spriteset for them
/:cl:

### CODE REVIEW

**Comment by Vivalas** [comment]
@Kmc2000 REEEEEEE merge conflicts.

**Comment by Vivalas** [comment]
You need to take the star trek sprites and merge them with the newniform sprites.

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -165,8 +206,8 @@
 	item_color = "green_suit"
 	can_adjust = 0
 
-/obj/item/clothing/under/gimmick/rank/executive_officer/suit
-	name = "executive officer's suit"
+/obj/item/clothing/under/gimmick/rank/head_of_personnel/suit
```
Comment (Vivalas) [inline code/modules/clothing/under/miscellaneous.dm]: Fix please.

File: code/game/objects/structures/door_assembly.dm [hunk 2]
```diff
@@ -332,6 +332,19 @@
 	anchored = 1
 	state = 1
 
+
+/obj/structure/door_assembly/door_assembly_bridg
```
Comment (Vivalas) [inline code/game/objects/structures/door_assembly.dm]: Is this "bridg" stuff intentional?

Comment (Kmc2000) [inline code/game/objects/structures/door_assembly.dm]: Yeah all the other door types are...ABRIDGED

File: code/modules/clothing/under/miscellaneous.dm (hunk 1 above)
Comment (Vivalas) [inline code/modules/clothing/under/miscellaneous.dm]: It should be XO.

**Pull request merged**